InteractiveHtmlBom/Run.bat

```diff
@@ -0,0 +1,37 @@
+@echo off
+
+echo -------------------------------------------------------------------------------------------------------------------
+echo -------------------------------------------------------------------------------------------------------------------
+echo                                                                                                                                                                                    -
+echo                                  Thankyou For Using Generate InteractiveBom
+echo                                              Powered By Qu1ck
+echo                                Bat Version : Powered By Scarrrr0725
+echo                                                                                                                                                                                    -
+echo --------------------------------------------------------------------------------------------------------------------
+echo --------------------------------------------------------------------------------------------------------------------
+
+::delete --show-dialog after frist sart up and setting
+set option=--show-dialog
+
+set FilePath=%~dp0
+set pyFilePath=%FilePath%generate_interactive_bom.py
+
+:convert
+set /p pathofEDASourceFile=Please Drag the EasyEDA PCB source file here :
+
+echo  Converting. . . . . . . . . .
+python %pyFilePath% %pathofEDASourceFile% %option%
+echo %pyFilePath% %pathofEDASourceFile%%option%
+
+echo -------------------------------------------------------------------------------------------------------------------
+echo -------------------------------------------------------------------------------------------------------------------
+echo                                                                                                                                                                                    -
+echo                              EDA Source File is Converted  to Bom Successfully ! ! ! !
+echo                                    Thankyou For Using Generate InteractiveBom
```

Comment:
Lets not be overly excited :)

"EDA source file is converted to bom successfully!"

No need for thanking user again, we did that in the intro banner.

Reply:
haha ,sorry for my english, again